### DISCUSSION TITLE
Cannot start launch.bat / Error Windows cannot find SharpMonoInjector.exe

### DISCUSSION THREAD

**Discussion by artefakta** [winstxnhdw/lc-hax#71] [Q&A]
Error Windows cannot find SharpMonoInjector.exe
I would appreciate a little help

**Comment by winstxnhdw** [comment]
Please look at the countless other discussions regarding your issue. Thank you.

**Reply by artefakta** [reply]
Thank you for your answer, but how can i find it. I'm really new to this but i learn fast.

**Reply by winstxnhdw** [reply]
https://github.com/winstxnhdw/lc-hax/discussions
